src/Domain/Model/Enum/AvailableLocales.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Model\Enum;
+
+use MabeEnum\Enum;
+
+/**
+ * The different category names.
+ */
+class AvailableLocales extends Enum
+{
+    public const fr = 'Français';
+    public const en = 'Anglais';
+    public const br = 'Breton';
+
+    /**
+     * @return array<string, string>
+     */
+    public static function getChoices(): array
+    {
+        return array_flip(self::getConstants());
+    }
+
+    public static function getDefault(): string
+    {
+        return 'fr';
```

Comment:
Prefer configuration to define the default locale : https://symfony.com/doc/current/configuration.html
I would go for an env variable here.

You can then use it here :
https://symfony.com/doc/current/translation/locale.html#setting-a-default-locale

This `getDefault` method probably wont be needed anymore...

The important thing being able to set the default locale in Symfony config :
https://symfony.com/doc/current/translation/locale.html#setting-a-default-locale

... to go further, Instead of this enum, I'll probably use a config for available locales (or supported locales) as well.
> see an example there https://symfony.com/doc/current/configuration.html#configuration-parameters but env would be even better.
> and there https://symfony.com/doc/current/translation/locale.html#the-locale-and-the-url for request validation

@JalilArfaoui stop me plz I'm going too far againnnnnn :scream:

Reply:
Also for evrything you may need to translante or deduce etc...
> https://symfony.com/doc/current/components/intl.html#locales

Reply:
hum... too far you said ?

Reply:
I have mixed feelings here …

Application default locale is not necessarily the same thing as contributor default locale. We could default to french for our backend + admin, and set the default locale as en_US for contributors … But it can still be defined in an env variable. Not mandatory for me though, since this is not something that would change regularly or by depends on environment …

About the enum, I’m really not a fan of `marc-mabe/php-enum` at all, and more generally of libraries trying to add such languages features to languages that do not support them and are not made to be used with them. It feels very clumsy and unnatural to me. But @gregoirelacoste tried to follow the existing code convention we have for `src/Domain/Model/Enum/CategoryName.php` so … I guess I could let it go like this for now, and have a refacto PR later that would replace the whole `marc-mabe/php-enum` with more PHP idiomatic code.

Reply:
exactly i followed the existing code convention. We are together agree with @lutangar  to use env var and params to configure locale even if I understand your point. But I think that now we need to finish and merge this issue, we wait since a bit too long